etc/scripts/generate-github-test-data.py

```diff
@@ -0,0 +1,140 @@
+# HOW TO USE ME?
+# `python3 /Users/jkay/repos/github-for-jira/etc/scripts/generate-github-test-data.py
+# Generate a personal access token from GitHub(PAT)
+# replace ACCESS_TOKEN with the PAT
+# replace USERNAME with GitHub username
+#
+import requests
+import os
+import subprocess
+import time
+import random
+
+# GitHub API base URL
+BASE_URL = 'https://api.github.com'
+
+# GitHub access token
+ACCESS_TOKEN = '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'
+
+# GitHub username
+USERNAME = 'joshkay10'
+
+# Number of branches and loop iterations
+NUM_BRANCHES = 5
+LOOP_ITERATIONS = 3
+
+# Helper function to make authenticated API requests
+def make_api_request(method, url, headers=None, data=None):
+    headers = headers or {}
+    headers['Authorization'] = f'Bearer {ACCESS_TOKEN}'
+    response = requests.request(method, url, headers=headers, json=data)
+    try:
+        response.raise_for_status()
+        return response.json()
+    except requests.exceptions.HTTPError as e:
+        print(f'Error making API request: {e}')
+        print(f'Response: {response.text}')
+        raise
+
+# Helper function to execute bash commands
+def run_bash_command(command):
+    process = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    stdout, stderr = process.communicate()
+    return process.returncode, stdout, stderr
+
+# Generate a unique repository name based on epoch time
+repo_name = f'repo-{int(time.time())}'
```

Comment:
Can we convert this to a function? Something like:

```
def generate_repo_name():
    return f'repo-{int(time.time())}
```

And then define repo_name in the top of the try block: `repo_name = generate_repo_name()`